README.md

```diff
@@ -7,14 +7,26 @@ This section assumes that you have at least one AntidoteDB node running. Check t
 
 The `antidotec_pb_socket` has several connection functions, but the simplest is `antidotec_pb_socket:start_link/2`, which takes in an address and port number and returns a connection wrapped in an Erlang Pid:
 
-```erl-sh
+```erl
 Pid = antidotec_pb_socket:start_link("127.0.0.1", 8087).
 ```
 
 If you are running a local instance of AntidoteDB, the default protocol buffers port number is `8087`.
 
 Once you have acquired a `Pid` connection to an AntidoteDB node, you are ready to run transactions.
 
+## Management Interface
+
+For coordinating Antidote instances, you can use the following operations via the Erlang client:
+
+```erl
+Pid = antidotec_pb_socket:start_link("127.0.0.1", 8087).
+%%
+ok = antidotec_pb_management:create_dc(Pid, ["node1", "node2"]).
```

Comment:
Here you are using Strings, but the type signature below says it should be `[node()]`. I think the type signature is too strict.

Reply:
No, the parameters were wrong.